service/src/ocall_bridge/bridge_api.rs

```diff
@@ -174,11 +174,9 @@ pub trait RemoteAttestationBridge {
 pub trait WorkerOnChainBridge {
 	fn worker_request(&self, request: Vec<u8>) -> OCallBridgeResult<Vec<u8>>;
 
-	fn send_block_and_confirmation(
-		&self,
-		confirmations: Vec<u8>,
-		signed_blocks: Vec<u8>,
-	) -> OCallBridgeResult<()>;
+	fn send_sidechain_blocks(&self, signed_blocks: Vec<u8>) -> OCallBridgeResult<()>;
+
+	fn send_confirmations(&self, confirmations: Vec<u8>) -> OCallBridgeResult<()>;
```

Comment:
Renaming suggestion (@clangenb may correct me here): `send_confirmations_to_layer_one` or something like that..?

Reply:
We decided not to use `layer_one` anymore but rather relative naming as there might be multiple hierarchies, so I'd suggest:

`send_to_parentchain(&self, extrinsics_vec_encoded)`

The current naming is actually wrong. The 'confirmations' may also contain callbacks from the trustedcall's execution.

Reply:
👍  I will re-name the function